### ISSUE TITLE
CDNO hierarchy doesn't have dietary nutritional component under material entity?

### ISSUE THREAD

**Issue by ddooley** [Southern-Cross-Plant-Science/cdno#69]
Was there a technical/logic issue trying to position the hierarchy? I see:

![image](https://user-images.githubusercontent.com/4000582/122877146-2122ac00-d2eb-11eb-9e20-30009498edfb.png)

**Comment by LilyAndres** [comment]
@ddooley yes it is a subclass of material entity, but it doesn't appear in the hierarchy as we want it to "remove" the BFO hierarchies so it's clearer for a more general audience. What you think?

**Comment by kaiiam** [comment]
I see @ddooley's point though it's technically under material entity. We bent over backwards to make a simpler top level hierarchy for CDNO as to simplify the experience for users but we want it to follow BFO even if hiding it. Not sure what's best here originally material entity was `plant` or similar as it's used in the concentration of pattern, but to make it more general purpose we had it be 'material entity' instead. 🤷 not sure what's best.

**Comment by LilyAndres** [comment]
@kaiiam, you mean that right now is not correct to have both the dietary nutritional component and concentration class  under material entity?

**Comment by kaiiam** [comment]
Technically `dietary nutritional component` should be subclass to `material entity`. Concentration shouldn't because it's a quality.

**Comment by ddooley** [comment]
All this becomes much easier under [COB](https://www.ebi.ac.uk/ols/ontologies/cob) - since COB's top level classes are much easier to comprehend, one design reason for it being promoted as new alternative in OBO for BFO.  You could consider moving dietary nutritional component under COB **material entity** once COB becomes more than just heir-apparent?  And COB has top level **characteristic/quality** path that would be the natural home for concentrations.

**Comment by LilyAndres** [comment]
@ddooley interesting, I will read a bit more about COB and see if there are any current ontologies that are starting to use COB instead of BFO.

**Comment by kaiiam** [comment]
Thanks @ddooley yes COB is a nice simplification of BFO I like what they did there I'd be in favor of that, but this was really due to @CropStoreDb's request to simplify the upper level, which we managed to do but is causing some issues like this, and more importantly not having the system be automatically reasoned under BFO so we might be creating some reasoning issues for others importing CDNO. 

Unfortunately I've never gotten COB to import fully using the regular import stack, I'm not sure what the issue there is.

**Comment by ddooley** [comment]
Attend the COB workshop at ICBO 2021 to see where it is headed!

**Comment by ddooley** [comment]
In terms of material entity etc, COB will just be providing the BFO term where semantics are the same.  So all you have to do in near future to be both "COB" and BFO compatible is to import "material entity" and position "dietary nutritional component" under it.  There will need to be a separate top level "role" for CHEBI roles like "water soluble vitamin" tho.

**Comment by LilyAndres** [comment]
Yes, ok @kaiiam explained how we could benefit from COB, I will try to work on this asap.

**Comment by kaiiam** [comment]
Yes I agree we should do this. @LilyAndres can you fix this in the upcoming release? In theory assuming there are no more role classes, dietary nutritional component can just be added as subclass to material entity.
